Aspire Budgeting/ViewModels/FileSelectorViewModel.swift

```diff
@@ -0,0 +1,51 @@
+//
+// FileSelectorViewModel.swift
+// Aspire Budgeting
+//
+
+import Foundation
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/ff44934e95fcfbf9892d0e49040b4320aa9b4509/Aspire Budgeting/ViewModels/FileSelectorViewModel.swift#L8"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>FileSelectorViewModel.swift:8</code></td>
    </tr>
  </tbody>
</table>